broker/service/job_cleaner_daemon.py

```diff
@@ -0,0 +1,57 @@
+import time
+import threading
+
+from broker.utils.linkedlist import LinkedList
+
+
+class JobCleanerDaemon():
+
+    def __init__(self, submissions):
+        self.submissions = submissions
+        self.queue = LinkedList()
+        self.thread = None
+        self.active = False
+
+    def start_delete_resources_management(self):
+        while not self.queue.is_empty():
+            time.sleep(1)
+            self.queue.head.value.remaining_time -= 1
+            if self.queue.head.value.remaining_time <= 0:
+                jobs_finished_ids = self.queue.pop().value.get_app_id()
+                for job_id in jobs_finished_ids:
```

Comment:
job_finished_ids is a result of a get_app_id(), so it is an id and not an array.

Reply:
> Wouldnt the best solution would be using python queue, or even the deque?

Is not possible because we need to insert elements using a custom strategy and the python queue offers operations only at its head and tail.

Reply:
> job_finished_ids is a result of a get_app_id(), so it is an id and not an array.

Modified to app_ids and get_app_ids()